database/mysql-ob/Chart.yaml

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+# Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
No need to update the header since this is an existing file

Reply:
Fixed with 0a68c76.